plugins/app-backend/src/service/router.ts

```diff
@@ -134,15 +146,130 @@ export async function createRouter(
 
     injectedConfigPath = await injectConfig({ appConfigs, logger, staticDir });
   }
+  const appConfigs = disableConfigInjection
+    ? await readConfigs({
+        config,
+        appDistDir,
+        env: process.env,
+      })
+    : undefined;
+
+  const assetStore =
+    options.database && !disableStaticFallbackCache
+      ? await StaticAssetsStore.create({
+          logger,
+          database: options.database,
+        })
+      : undefined;
 
   const router = Router();
 
   router.use(helmet.frameguard({ action: 'deny' }));
 
+  const publicDistDir = resolvePath(appDistDir, 'public');
+
+  const enablePublicEntryPoint =
+    (await fs.pathExists(publicDistDir)) && auth && httpAuth;
+
+  if (enablePublicEntryPoint && auth && httpAuth) {
+    const publicRouter = Router();
+
+    publicRouter.use(createCookieAuthRefreshMiddleware({ auth, httpAuth }));
```

Comment:
We could skip this one now right? Relying on it being added by the http router service instead

Reply:
Yeap